versions/1/data_stream/spec.yml

```diff
@@ -3,7 +3,7 @@ spec:
   contents:
   - description: Folder containing a single data stream definition
     type: folder
-    pattern: '[a-z0-9]([a-z0-9_]+[a-z0-9])?'
+    pattern: '([a-z0-9][a-z0-9]|[a-z0-9][a-z0-9_][a-z0-9_]+)'
```

Comment:
Why was this change necessary?

Reply:
`pe` datastream was not valid.

Reply:
BTW I changed it now to `'([a-z0-9][a-z0-9]|[a-z0-9][a-z0-9_]+[a-z0-9])'

Reply:
Understood. BTW, you can change the first option in the regex (the left side of the `|`) to `[a-z0-9]{2}`.

Reply:
Nice, I simplified the regular expression.